### ISSUE TITLE
MAC version

### ISSUE THREAD

**Issue by ClickNciak** [bsondermann/BlackboxSticksExporter3D#2]
It's possible to have a version for MAC ?
best regards
diwellington

**Comment by bsondermann** [comment]
Hi, sadly I dont have a mac machine here and so I cant compile a mac version. If somebody wants to compile it I would add it to the assets in the next version.

**Comment by ClickNciak** [comment]
> Hi, sadly I dont have a mac machine here and so I cant compile a mac version. If somebody wants to compile it I would add it to the assets in the next version.

Hy 
I'm not a programmer but I think is a really good feature !
Thx for your work!! 🙌🏽

**Comment by PenguinK** [comment]
> Hi, sadly I dont have a mac machine here and so I cant compile a mac version. If somebody wants to compile it I would add it to the assets in the next version.

hello ,May be I can help compile Mac verison.It is coding in JAVA？and I'don know how to open .pde file.May be you can teach me how to compile it my email is Penguinmore@gmail.com

**Comment by bsondermann** [comment]
Hi, thanks for the offer but I found out that some of the resources are not available for mac so unfortunately I dont think that there will be a mac version :(

**Comment by PenguinK** [comment]
Is ffmpeg？or something？

bsondermann <notifications@github.com> 于 2021年1月16日周六 08:33写道：

**Comment by bsondermann** [comment]
Yeah ffmpeg and blackbox decoder but I think that those would be available for mac too. The problem is blender because as far as I know the mac version of blender doesn't support command line inputs

**Comment by PenguinK** [comment]
Sorry,I have never use blender before.I googled "blender command line mac"
and found this
https://blender.stackexchange.com/questions/2078/how-to-use-blender-command-lines-in-osx
I don't know it is useful.
You blackbox is awesome ,really want it run on Mac os

bsondermann <notifications@github.com> 于 2021年1月16日周六 21:08写道：
